AnimalPicker.java

```diff
@@ -5,14 +5,17 @@ public static void main (String args[]) {
         boolean valid = false;
         
         while (valid == false) {
-            System.out.println("\nWould you like to see a cat or a dog?");
+            System.out.println("\nWould you like to see a cat, dog or fish?");
             String choice = scan.nextLine();
             if (choice.equalsIgnoreCase ("cat")) {
                 System.out.println( "/\\__/\\\n( o.o )\n > ^ <");
                 valid = true;
             } else if (choice.equalsIgnoreCase("dog")) {
                 System.out.println("  __      _\no'')}____//\n`_/      )\n(_(_/-(_/");
                 valid = true;
+            } else if (choice.equalsIgnoreCase("fish")) {
```

Comment:
I like the use of the equalsIgnoreCase() function for string comparisson. I used equals() which takes case into account.